test/test_impl.py

```diff
@@ -611,21 +611,3 @@ def test_version(self) -> None:
                     )
                 ],
             )
-
-
-class DummyModel(BaseModel):
-    base64_bytes: impl.Base64Bytes
-
-
-class TestBase64Bytes:
-    # See the docstrings for `_impl.Base64Bytes` for more details
-    def test_decoding(self) -> None:
-        # This raises when using our custom type. When using Pydantic's Base64Bytes,
-        # this succeeds
-        # The exception message is different in Python 3.9 vs >=3.10
-        with pytest.raises(ValueError, match="Non-base64 digit found|Only base64 data is allowed"):
-            DummyModel(base64_bytes=b"a\n\naaa")
-
-    def test_encoding(self) -> None:
-        model = DummyModel(base64_bytes=b"aaaa" * 76)
-        assert "\\n" not in model.model_dump_json()
```

Comment:
Can we keep this backstop test, but with the upstream `Base64Bytes`? I don't expect them to regress here, but it's a good regression test to have in case they do 🙂

Reply:
done! I only kept the encoding test, since it's the one we care about. The difference in behavior for decoding was only due to me using `b64decode(strict=True)`, which Pydantic doesn't.